From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Fri, 22 Apr 2016 08:46:56 -0400
Subject: fixup RELEASE

---
 configure/RELEASE | 19 +++++++------------
 1 file changed, 7 insertions(+), 12 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index 0e2c166..1a76b2c 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -19,19 +19,14 @@ SUPPORT=/home/oxygen/MOONEY/epics/synAppsSVN/support
 -include $(TOP)/../configure/SUPPORT.$(EPICS_HOST_ARCH)
 
 # Modules required for building module
-ASYN=$(SUPPORT)/asyn-4-23
-CALC=$(SUPPORT)/calc-3-4-1
-IP=$(SUPPORT)/ip-2-15
-STREAM=$(SUPPORT)/stream-2-6
-IPAC=$(SUPPORT)/ipac-2-12
+ASYN=$(EPICS_BASE)
+CALC=$(EPICS_BASE)
+#IP=$(EPICS_BASE)
+STREAM=$(EPICS_BASE)
+#IPAC=$(EPICS_BASE)
 
 # Modules required for building module application
-AUTOSAVE=$(SUPPORT)/autosave-5-4-1
+AUTOSAVE=$(EPICS_BASE)
 
 # EPICS_BASE usually appears last so other apps can override stuff:
-EPICS_BASE=/home/oxygen/MOONEY/epics/bazaar/base-3.14
--include $(TOP)/../configure/EPICS_BASE.$(EPICS_HOST_ARCH)
-
-#Capfast users may need the following definitions
-#CAPFAST_TEMPLATES=
-#SCH2EDIF_PATH=
+EPICS_BASE=/usr/lib/epics
