From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Fri, 22 Apr 2016 08:59:11 -0400
Subject: cleanup const char* warning in DG645

---
 delaygenApp/src/drvAsynDG645.cpp | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/delaygenApp/src/drvAsynDG645.cpp b/delaygenApp/src/drvAsynDG645.cpp
index 950f416..dd3d0d1 100644
--- a/delaygenApp/src/drvAsynDG645.cpp
+++ b/delaygenApp/src/drvAsynDG645.cpp
@@ -100,7 +100,7 @@
 /* Forward struct declarations */
 typedef enum {Octet,Float64,Int32} ifaceType;
 
-static char *driver = "drvAsynDG645";      /* String for asynPrint */
+static const char *driver = "drvAsynDG645";      /* String for asynPrint */
 
 
 /* Declare port driver structure */
@@ -129,21 +129,21 @@ struct Port
 struct Status
 {
   int code;
-  char* msg;
+  const char* msg;
 };
 
 
 /* Declare command structure */
 struct Command
 {
-  char* readCommand;
+  const char* readCommand;
   asynStatus (*readFunc)(int which,Port *pport,char* inpBuf,int inputSize,int* eomReason,ifaceType asynIface);
   int (*readConv)(int which,Port *pport,char* inpBuf,int maxchars,void* outBuf,ifaceType asynIface);
 
-  char* writeCommand;
+  const char* writeCommand;
   asynStatus (*writeFunc)(int which, Port *pport,void* data,ifaceType asynIface);
 
-  char *tag;
+  const char *tag;
   //  char* desc; // not used right now, so they are commented out
 };
 
@@ -182,8 +182,8 @@ static asynStatus readOctet(void* ppvt,asynUser* pasynUser,char* data,size_t max
 static asynOctet ifaceOctet = { writeOctet, readOctet, flushOctet};
 
 /* Forward references for external asynOctet interface */
-static asynStatus writeOnly(Port* pport,char* outBuf);
-static asynStatus writeRead(Port* pport,char* outBuf,char* inpBuf,int inputSize,int *eomReason);
+static asynStatus writeOnly(Port* pport,const char* outBuf);
+static asynStatus writeRead(Port* pport,const char* outBuf,char* inpBuf,int inputSize,int *eomReason);
 
 
 /* Forward references for command table methods */
@@ -584,7 +584,7 @@ static int cvtCopyText(int which, Port *pport,char* inpBuf,int maxchars,void* ou
 static int cvtErrorText(int which, Port *pport,char* inpBuf,int maxchars,void* outBuf,ifaceType asynIface)
 {
   int code;
-  char *m = NULL;
+  const char *m = NULL;
 
   code=atoi(inpBuf);
   for(int i=0; i<statusLen; i++) 
@@ -648,7 +648,7 @@ static int cvtChanRef(int which, Port *pport,char* inpBuf,int maxchars,void* out
 }
 static int cvtChanDelay(int which, Port *pport,char* inpBuf,int maxchars,void* outBuf,ifaceType asynIface)
 {
-  char* delayText[] = {"T0", "T1", "A", "B", "C", "D", "E", "F", "G", "H"};
+  const char* delayText[] = {"T0", "T1", "A", "B", "C", "D", "E", "F", "G", "H"};
 
   int chan,siz;
   epicsFloat64 delay;
@@ -949,7 +949,7 @@ static asynStatus readOctet(void* ppvt,asynUser* pasynUser,char* data,size_t max
 /****************************************************************************
  * Define private DG645 external interface asynOctet methods
  ****************************************************************************/
-static asynStatus writeOnly(Port* pport,char* outBuf)
+static asynStatus writeOnly(Port* pport,const char* outBuf)
 {
   asynStatus status;
   size_t nActual,nRequested=strlen(outBuf);
@@ -969,7 +969,7 @@ static asynStatus writeOnly(Port* pport,char* outBuf)
 
   return status;
 }
-static asynStatus writeRead(Port* pport,char* outBuf,char* inpBuf,int inputSize,int* eomReason)
+static asynStatus writeRead(Port* pport,const char* outBuf,char* inpBuf,int inputSize,int* eomReason)
 {
   asynStatus status;
   size_t nWrite,nRead,nWriteRequested=strlen(outBuf);
